chart/templates/tests/test-up-timescaledb.yaml

```diff
@@ -0,0 +1,126 @@
+{{- $tsdb := index .Values "timescaledb-single" -}}
+{{- if $tsdb.enabled -}}
+---
+apiVersion: v1
+kind: ServiceAccount
+automountServiceAccountToken: false
+metadata:
+  name: "{{ .Release.Name }}-test-up-timescale"
+  namespace: "{{ template "tobs.namespace" . }}"
+  labels:
+    {{- include "tobs.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: "{{ .Release.Name }}-test-up-timescale"
+  namespace: "{{ template "tobs.namespace" . }}"
+  labels:
+    {{- include "tobs.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: "{{ .Release.Name }}-test-up-timescale"
+subjects:
+  - kind: ServiceAccount
+    name: "{{ .Release.Name }}-test-up-timescale"
+    namespace: "{{ template "tobs.namespace" . }}"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: "{{ .Release.Name }}-test-up-timescale"
+  namespace: "{{ template "tobs.namespace" . }}"
+  labels:
+    {{- include "tobs.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmap
+  verbs:
+  - get
+  resourceNames:
+  - ""
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ .Release.Name }}-test-up-timescale"
+  namespace: "{{ template "tobs.namespace" . }}"
+  labels:
+    {{- include "tobs.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+data:
+  test-timescale.sh: |-
+    #!/bin/bash
+
+    #set -euo pipefail
+    set -o pipefail
+    #set -x
+
+    # Setup query to prometheus
+    RELEASE="{{ .Release.Name }}"
+    NAMESPACE="{{ .Release.Namespace }}"
+
+    PROM_QUERY_URL="http://${RELEASE}-promscale.${NAMESPACE}.svc:9201/api/v1/query?"
+
+    function query() {
+      local query
+      query=$(cat <<-EOF
+    query=up==0
```

Comment:
Do you think we could have here also a test for `prometheus_build_info` metric existence or do you think we should create a separate PR for that?

Reply:
Just saw I never answered this.  I was going to suggest just creating a new PR for this.  I'll go ahead and create the issue.